card/db/migrate_core_cards/20141119001955_make_symlinks_relative.rb

```diff
@@ -2,7 +2,7 @@
 
 class MakeSymlinksRelative < Card::Migration
   def up
-    Cardio.paths['files'].each do |file_path| 
+    Card.paths['files'].each do |file_path| 
```

Comment:
Lot's of changes here are just this change that you requested, but card-gem was already way to unweildy.  Card.(X) are just aliases for the other, and sometime we still have to use Cardio when Card is not yet loaded (most migrations and tasks).